README.md

```diff
@@ -21,7 +21,7 @@ We've applied the philosophy of _"do one thing and do it well"_, we are focused
 * Nano utilizes a novel [block-lattice](https://github.com/nanocurrency/raiblocks/wiki/Block-lattice) architecture, unlike conventional blockchains used in many other cryptocurrencies.
 * The network requires minimal resources, no high-power mining hardware, and can process high transaction throughput.
 * Offers instantaneous transactions with zero fees and unlimited scalability, making Nano an ideal solution for peer-to-peer transactions.
-* As of May 2018, the Nano network has processed over ten million transactions with an unpruned ledger size of only 3.8GB.
+* As of August 2018, the Nano network has processed over twelve million transactions with an unpruned ledger size of only [4.4GB](https://nano.meltingice.net/status).
```

Comment:
@nanotroy Can you check this out ?

Reply:
Database size is currently 4.67 GB, 12.8 Million blocks. Should blocks replace transactions due to Send/Receives?